en/docs/reference/configs.json

```diff
@@ -1,2413 +0,0 @@
-{
-    "configs": [
```

Comment:
@ruthryi , is it correct that this file has been deleted?

Reply:
Yes. This is redundant. The configs.json in the 'tools' folder is the one that should be updated.